0. Kirish/0.1 Maqsad, Modellashtirish va Simulyatsiya.md

```diff
@@ -0,0 +1,22 @@
+# 0.1 Maqsad, Modellashtirish va Simulyatsiya
+
+## 0.1.0 Kirish
+
+"Meyor" dasturiy ta’minot tuzilishini uch asosiy yo'nalishda ifodalaydi: Maqsad, Modellashtirish va Simulyatsiya. Ularning har biri muhandislarning muvaffaqiyatli yechim va aniq maqsadga erishishlarida muhim rol o'ynashdi. Bu yo'nalishlarning bajarilish ketma-ketligi ham o'zaro bog'liqdir. Maqsad Modellashtirish jarayonida mavjud bo'lishi shart. Ayni paytda Simulyatsiya jariyoni ham modellarsiz bajarila olmaydi. Bunday ketma-ketlik muhandislik faoliyati boshlanishida hal qiluvchi ma’no kasb etsadi, shuni anglash kerakki, jarayonning o'zi tanlab takrorlanadi. Maqsaddagi o'zgarish Simulyatsiyada o'z aksini topishi mumkin, ammo Modelni o'zgartirish shart bo'lmaydi. Modeldagi o'zgarish na Maqsad va na Simulyatsiyani o'zgartirishini taqazo qilishi mumkin.
+
+<br />
+    <div align=center>
+        <img src="./Resurslar/Maqsad, Modellashtirish va Simulyatsiya/maqsad,modellashtirish va simulyatsiya.png" />
+    </div>
+<br />
+
+
+## 0.1.1 Maqsad
+
+Maqsad qo'yish jarayoni - nima uchun bizga yechim kerakligini aniqlash qobiliyatidir. Misol uchun, ma'lum bir oziq-ovqat do'kon javonida qancha mahsulot borligini bilish bilan bog'liq muammo bor. Biz qo'lda hisoblash jarayonini samarasiz deb hisoblaymiz va mahsulotlarni to'g'ri hisoblashimiz uchun tizimni joriy qilish kerak.
+
+Mulohaza yuritish ko'p jihatdan muammolarni kuzatish va keyin ushbu muammoni hal qiladigan yechimni ishlab chiqish uchun muammoni ifodalash qobiliyatiga bog'liq.
```

Comment:
Bu gap qolib ketibti:

`Purposing, therefore, is to find a reason to take action.`

Reply:
uzr e’tibor bermagan ekanman tog’irlab boshqatdan PR tashaymi